pkg/server/internal/packetio.go

```diff
@@ -150,19 +150,21 @@ func (p *PacketIO) readOnePacket() ([]byte, error) {
 		if _, err := io.ReadFull(r, header[:]); err != nil {
 			return nil, errors.Trace(err)
 		}
+
+		// To be compatible with MariaDB Connector/J 2.x, 
```

Comment:
```suggestion
		// To be compatible with MariaDB Connector/J 2.x,
```

please format this file using your IDE or `make fmt`

Reply:
Done. Sorry, forgot to fmt the code